nova/tests/unit/virt/vmwareapi/test_vmops.py

```diff
@@ -502,25 +513,36 @@ def test_unrescue_power_on(self):
     def test_unrescue_power_off(self):
         self._test_unrescue(False)
 
-    def _test_unrescue(self, power_on):
+    def test_unrescue_efi(self):
+        self._test_unrescue(False, True)
```

Comment:
```suggestion
        self._test_unrescue(power_on=False, efi=True)
```

Could also fix it above, if it's not too disjoint for you.

Reply:
"Please change more upstream code than necessary"? I don't like that.